qemu.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = QEMU
+Version = 2.7.0
+License = GPL v2
+Description = An open source and very fast machine emulator/virtualizer. It can emulate processors based on many various architectures (eg. x86, ARM, MIPS, PowerPC). Run with -display sdl flag to avoid the display glitches.
```

Comment:
Suggestion: You could use double quotes for `"-display sdl"`.

Reply:
Done.